table/tables/mutation_checker.go

```diff
@@ -175,7 +175,8 @@ func checkHandleConsistency(rowInsertion mutation, indexMutations []mutation, in
 // To check (1), we need
 // (a) {added indices} is a subset of {needed indices} => each index mutation is consistent with the input/row key/value
 // (b) {needed indices} is a subset of {added indices}. The check process would be exactly the same with how we generate
-// 		the mutations, thus ignored.
+//
```

Comment:
Please move the words in line 179 to 177.

Reply:
Fix